provider/aws/formation/rack.json

```diff
@@ -1722,6 +1734,49 @@
         }
       }
     },
+    "ScheduledActionDown":{
+      "Type":"AWS::AutoScaling::ScheduledAction",
+      "Condition": "SetScheduledActionDown",
+      "Properties":{
+        "AutoScalingGroupName": { "Ref": "Instances" },
```

Comment:
should we add `"DesiredCapacity": "0",` to this one aswell?

Reply:
we have `"MaxSize": "0"` and `MinSize": "0"` set to zero, `DesiredCapacity` is not needed.